resources/lang/de/command/messages.php

```diff
@@ -15,28 +15,28 @@
         'confirm_delete' => 'Bist du dir wirklich sicher?',
         'no_users_found' => 'Es wurden keine User gefunden.',
         'multiple_found' => 'Es wurden mehrere Accounts gefunden.',
-        'ask_admin' => 'Is this user an administrator?',
+        'ask_admin' => 'Ist dieser User ein Administrator?',
```

Comment:
I think Ist dieser Benutzer ein Administrator? would be better

Reply:
They used "User" all over the place in the original German translation, so I thought that the term "User" is what they wanted to use throughout the Panel.